### ISSUE TITLE
Analytics (Desktop) 

### ISSUE THREAD

**Issue by nbintertech** [ORNL-AMO/AMO-Tools-Desktop#6016]
Goals:

**1. Google Analytics**
- Understand what routes/pages of the app are visited and frequency

**2. Anonymized energy savings data should be collected**
- Start with Rollup report
- Notify user that anonymized data is being sent - present opt out options
- Data is piped to server where  a csv/report can be created from data

Lower priority
3. (Google Analytics part 2) - Which features are used (where are clicks happening) 

NOTE: ignore developers/Kristina/when in dev mode

**Comment by nbintertech** [comment]
Remove any /testing endpoint hits once this is merged, otherwise development users will still be logged.
